src/fixate/config/helper.py

```diff
@@ -60,3 +63,85 @@ def load_json_config(in_file, config_name=None):
         fixate.config.__dict__.update(json.load(in_file))
 
 
+def load_yaml_config(in_file, config_name=None) -> None:
+    """
+    :param in_file:
+     valid open file like such as
+        open(in_file) as f:
+            load_json_config(f)
+     or stringio instance
+     or path to file
+    :param config_name:
+     optional way of grouping the config details
+    :usage
+     my_yaml_file
+     HI: WORLD
+     config_name = None
+     >>> import fixate.config
+     >>> with open("my_yaml_file") as f:
+     >>>    load_yaml_config("my_yaml_file")
+     >>> print(fixate.config.HI)
+     "WORLD"
+     config_name = "My_Json"
+     >>> import fixate.config
+     >>> with open("my_yaml_file") as f:
+     >>>    load_yaml_config("my_yaml_file", "My_Yaml")
+     >>> print(fixate.config.My_Yaml)
+     {"HI": "WORLD"}
+    """
+    yaml = ruamel.yaml.YAML(typ="safe", pure=True)
+    yaml.default_flow_style = False
+    if os.path.exists(in_file):
+        with open(in_file) as f:
+            if config_name:
+                fixate.config.__dict__.update({config_name: yaml.load(f)})
+            else:
+                fixate.config.__dict__.update(yaml.load(f))
+        return
+    if config_name:
+        fixate.config.__dict__.update({config_name: yaml.load(in_file)})
+    else:
+        fixate.config.__dict__.update(yaml.load(in_file))
```

Comment:
This is just importing yaml file except I kept the conventional API that The json loader and dictionary loader that already existed in that file. The extra boiler plate is so you can pass a path instead of a file pointer